clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesKind.java

```diff
@@ -134,23 +194,26 @@ private static KubernetesKind createAndRegisterKind(
       @Nonnull String name,
       @Nonnull KubernetesApiGroup apiGroup,
       @Nullable String alias,
+      @Nullable String plural,
       boolean isNamespaced,
       boolean hasClusterRelationship) {
     return kindRegistry.registerKind(
         new KubernetesKind(
-            name, apiGroup, alias, isNamespaced, hasClusterRelationship, false, true));
+            name, apiGroup, alias, plural, isNamespaced, hasClusterRelationship, false, true));
   }
 
   private KubernetesKind(
       @Nonnull String name,
       @Nonnull KubernetesApiGroup apiGroup,
       @Nullable String alias,
+      @Nullable String plural,
```

Comment:
I worry about adding another field here to hold the plural version of a name; this is something else that we'll need to make sure stays in sync.  I think this will also break people who have registered CRDs in their clouddriver config, as they (presumably) have not added the plural version there so the kind will be registered with a null plural.

I think it would be more robust to use the separate `apigroup` and `kind` columns from the `api-resources` call and to compare base on `apiGroup` and `name`.

Reply:
Totally agree. The only reason I opted for adding the plurals is because it made parsing the result of `api-resources` easier due to it's semi-tabular nature and the fact that I can't just `-o json` the result. I'll see if there's a better way to tease the data out.